Add information relevant on Windows in the colcon tutorial (#1414)
ros2/ros2_documentation: source/Tutorials/Colcon-Tutorial.rst

To run tests for the packages we just built, run the following:

.. tabs::

  .. group-tab:: Linux

    .. code-block:: console

      colcon test

  .. group-tab:: macOS

    .. code-block:: console

      colcon test

  .. group-tab:: Windows

    Remember to use a ``x64 Native Tools Command Prompt for VS 2019`` for executing the following command, as we are going to build a workspace.

    .. code-block:: console

      colcon test --merge-install

>     You also need to specify ``--merge-install`` here if that was used when building.

Reviewer: ```suggestion
    You also need to specify ``--merge-install`` here since we used it for building above.
```
